libs/api-interfaces/src/lib/aggregate-views/view-models/resources/photograph.view-model.interface.ts

```diff
@@ -1,7 +1,7 @@
 import { IBaseViewModel } from '../base.view-model.interface';
 
 export interface IPhotographViewModel extends IBaseViewModel {
-    // Eventually, this should be a digital asset ID
+    // Eventually, this should be a digital asset ID- update this soon
```

Comment:
I don't know that it matters as long as the query service handles building the URL.

Reply:
The comment may simply apply to the `PhotographViewModel`.

Reply:
@yaanahuu2 This might be relevant to your PR.